deps/rabbit/priv/schema/rabbit.schema

```diff
@@ -1370,6 +1370,13 @@ fun(Conf) ->
     end
 end}.
 
+%% Register node during cluster formation when backend supports registration.
+%%
+
+{mapping, "cluster_formation.registration", "rabbit.cluster_formation.register", [
```

Comment:
I would prefer a name that implicitly tells the user it wants a boolean. For instance `register` (like application environment name you picked) or `perform_registration` because "register" is both a verb and a noun.